### ISSUE TITLE
[Permission] Porting to fabric

### ISSUE THREAD

**Issue by KirboSoftware** [Mechalopa/Hostile-Mobs-and-Girls#14]
Hello, I'm Joost aka JoostMSoftware and I'm making mainly fabric mods. 

Recently someone brought me attention to this lovely mod and I was wondering if it was for fabric (it was not) so I was wondering if I could get permission to make a public port for the fabric eco space. 

Hope you're having a lovely day and I look forward to your response!

Kind regards, 

Joost aka JoostMSoftware.

**Comment by Mechalopa** [comment]
Yes, you can create fabric port as long as you adhere to the license.

**Comment by KirboSoftware** [comment]
So I would have to follow the LGPLv3 license when making the port? And is there a way I could contact you besides this issue?

**Comment by Mechalopa** [comment]
Yes, it is. But to be honest,  I don't know much about many things and my time is limited so it is difficult for me to offer much support (sorry).
https://twitter.com/Mechalopa Twitter is here.

**Comment by KirboSoftware** [comment]
That's fair!

**Comment by night2078** [comment]
Not sure if this is the right place to ask for this, but are there any news regarding a fabric port? Didn't really find anything about this topic other than here.

**Comment by KirboSoftware** [comment]
someone else ported it seems like 🤷 I didn't have time to develop it and idk if the person who did port it, asked the owner for permission.

**Comment by night2078** [comment]
Oh okay. Well, i just can't find any port, so it might be hidden somewhere. Anyway, thanks for the response though, appreciate it.

**Comment by KirboSoftware** [comment]
wait nvm, I got confused with another mod with a similar name. But i could do it once I've more time available. 👍 @night2078
